include/megaapi.h

```diff
@@ -886,6 +886,19 @@ class MegaNode
         virtual std::string* getAttrString();
 
         /**
+         * @brief Returns a string that contains the file attributes related to the node
+         *
+         * The return value is only valid for nodes attached in a chatroom. In all other cases this function
+         * will return an empty string.
```

Comment:
The above is not always true, since there are more cases where the file attribute string will be copied from the original `Node`. However, only for attachments it's guaranteed that fileattribute is not empty.